db/seeds.rb

```diff
@@ -6,3 +6,16 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+if ENV['USE_DEMO_DATA'] || Settings.use_demo_data || Rails.env.test?
```

Comment:
This shouldn't be part of the seeds, as is not needed for a regular environment to work.

The seeds shouldn't be needed for the test environment. It should be the test responsibility to create the objects that it needs in order to run.

Please extract this part of the seeds into a rake task that does the same, and into the setup part of the tests that need this data.